src/components/navigation/header/config/DrawerNodeItems.tsx

```diff
@@ -1289,10 +1289,10 @@ export const drawerNodeItems = ({
         {
           translationKey: 'header.userMenu.contactDetails',
           link: {
-            de: '/de/member/masterdata/addressedit',
-            en: '/de/member/masterdata/addressedit',
-            fr: '/fr/member/masterdata/addressedit',
-            it: '/it/member/masterdata/addressedit',
+            de: '/de/profile',
+            en: '/de/profile',
+            fr: '/fr/profile',
+            it: '/it/profile',
```

Comment:
Can we simplify the configuration?

Before:
```ts
        {
          translationKey: 'header.userMenu.contactDetails',
          link: {
            de: '/de/profile',
            en: '/en/profile',
            fr: '/fr/profile',
            it: '/it/profile',
          },
          visibilitySettings: {
            userType: {
              private: true,
              professional: false,
            },
            brand: {
              autoscout24: true,
              motoscout24: true,
            },
          },
        },
        {
          translationKey: 'header.userMenu.contactDetails',
          link: {
            de: '/de/profile',
            en: '/de/profile',
            fr: '/fr/profile',
            it: '/it/profile',
          },
          visibilitySettings: {
            userType: {
              private: false,
              professional: true,
            },
            brand: {
              autoscout24: true,
              motoscout24: true,
            },
          },
        },
```

After:
```ts
        {
          translationKey: 'header.userMenu.contactDetails',
          link: {
            de: '/de/profile',
            en: '/en/profile',
            fr: '/fr/profile',
            it: '/it/profile',
          },
          visibilitySettings: {
            userType: {
              private: true,
              professional: true,
            },
            brand: {
              autoscout24: true,
              motoscout24: true,
            },
          },
        },
```

Reply:
Absolutely. It does make sense. Thanks!